blocks/gmo-program-details/gmo-program-details.js

```diff
@@ -309,14 +315,80 @@ async function addProgramStats(block) {
+        // }
+
+        platforms.forEach(async platform => {
+            let assetResult = {imageUrl : '', imageAltText: '', assetCount: 0};
+            assetResult = await searchAsset(programName, programName.campaignName, deliverableType, platform);
+            let assetCount = assetResult.assetCount;
+            // deliverableType, platform, assetCount
+            if (!deliverableTypeToPlatformsMap.has(deliverableType)) {
+                // Case: If the deliverable type is not in the map, create a new entry with the platform added to the new map.
+                const deliverableTypesPlatformAssetCountMap = new Map();
+                deliverableTypesPlatformAssetCountMap.set(platform, assetCount);
+                // Create a new entry for each deliverable type with the new map.
+                deliverableTypeToPlatformsMap.set(deliverableType, deliverableTypesPlatformAssetCountMap);
+            } else {
+                // Case: If the deliverable type is in the map, add the platform to the existing set.
+                const deliverableTypesPlatformAssetCountMap = deliverableTypeToPlatformsMap.get(deliverableType);
+                deliverableTypesPlatformAssetCountMap.set(platform, assetCount);
+                deliverableTypeToPlatformsMap.set(deliverableType, deliverableTypesPlatformAssetCountMap);
+            }
+
+            if (!platformToDeliverableTypesMap.has(platform)) {
+                // Case: If the platform is not in the map, create a new entry with the deliverableType added to the new map.
+                const platformdeliverableTypesAssetCountMap = new Map();
+                platformdeliverableTypesAssetCountMap.set(deliverableType, assetCount);
+                // Create a new entry for each platform with the new map.
+                platformToDeliverableTypesMap.set(platform, platformdeliverableTypesAssetCountMap);
+            } else {
+                // Case: If the platform is in the map, add the deliverableType to the existing set.
+                const platformdeliverableTypesAssetCountMap = platformToDeliverableTypesMap.get(platform);
+                platformdeliverableTypesAssetCountMap.set(deliverableType, assetCount);
+                platformToDeliverableTypesMap.set(platform, platformdeliverableTypesAssetCountMap);
+            }
+
+            // if (!platformToDeliverableTypesMap.has(platform)) {
+            //     const deliverableTypesPlatformAssetCountMap = new Map();
+            //     deliverableTypesPlatformAssetCountMap.set(platform, assetResult.assetCount);
+            //     // Create a new set for each platform with the deliverable type added to the new array.
+            //     platformToDeliverableTypesMap.set(platform, new Array());
+            // }
+            // platformToDeliverableTypesMap.get(platform).push(deliverableType);
+        });
+
+    });
+
+    console.log('Deliverable Type to Platforms Map:', deliverableTypeToPlatformsMap);
```

Comment:
Would suggest pulling the console statements out also unless absolutely necessary